packages/espressocash_api/lib/src/dto/withdraw.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:espressocash_api/espressocash_api.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'withdraw.freezed.dart';
+part 'withdraw.g.dart';
+
+@freezed
+class WithdrawPaymentRequestDto with _$WithdrawPaymentRequestDto {
+  const factory WithdrawPaymentRequestDto({
+    required String senderAccount,
+    required String receiverAccount,
```

Comment:
Let's rather have specific endpoints for now – this one for scalex. Different partners have different flows anyway, so we don't know for sure if this structure is universal. And for scalex, I think, we can just pass `partnerOrderId` and call the API on the backend to get all the details.